app/views/take_whole_pot_calculator/_results.html.erb

```diff
@@ -0,0 +1,23 @@
+<p>
+  Based on what you've told us, if you take your whole <%= number_to_currency(@pot, unit: '£') %>
```

Comment:
How about introducing an `en-GB` locale so that we don’t have to specify the currency unit each time?